Intersect.Client.Core/Interface/Game/Spells/SpellsWindow.cs

```diff
@@ -74,21 +74,13 @@ private void InitItemContainer()
     public override void Hide()
     {
         _contextMenu?.Close();
-        base.Hide();
-    }
 
-    public FloatRect RenderBounds()
-    {
-        var rect = new FloatRect()
+        // dont hide window if we are dragging something
+        if (Items.Any(c => c.IconImage == DragAndDrop.CurrentPackage?.DrawControl))
```

Comment:
Same comment as I had in the other windows (I added a longer message in `InventoryWindow` though)